keyboards/owlab/suit80/iso/info.json

```
@@ -0,0 +1,194 @@
+{
+    "keyboard_name": "suit80",
+    "url": "https://www.qwertykeys.com/products/jrrgvdsyu47wer",
```

```suggestion
    "url": "",
```
Another URL pointing to Jelly Epoch 75%.